tests/resolvers.test.ts

```
@@ -85,27 +89,31 @@ describe('Query Resolvers', async () => {
   let zkApp: HelloWorld;
 
   before(async () => {
-    setNetworkConfig();
+    try {
+      setNetworkConfig();
 
-    const schema = createSchema({
-      typeDefs: loadSchemaSync('./schema.graphql', {
-        loaders: [new GraphQLFileLoader()],
-      }),
-      resolvers,
-    });
-    const context = await buildContext(PG_CONN);
-    const yoga = createYoga<GraphQLContext>({ schema, context });
-    executor = buildHTTPExecutor({
-      fetch: yoga.fetch,
-    });
+      const schema = createSchema({
+        typeDefs: loadSchemaSync('./schema.graphql', {
+          loaders: [new GraphQLFileLoader()],
+        }),
+        resolvers,
+      });
+      const context = await buildContext(PG_CONN);
+      const yoga = createYoga<GraphQLContext>({ schema, context });
+      executor = buildHTTPExecutor({
+        fetch: yoga.fetch,
+      });
 
-    zkAppKeypair = await Lightnet.acquireKeyPair();
-    senderKeypair = await Lightnet.acquireKeyPair();
-    zkApp = await deployContract(
-      zkAppKeypair,
-      senderKeypair,
-      /* fundNewAccount = */ false
-    );
+      zkAppKeypair = await Lightnet.acquireKeyPair();
+      senderKeypair = await Lightnet.acquireKeyPair();
+      zkApp = await deployContract(
+        zkAppKeypair,
+        senderKeypair,
+        /* fundNewAccount = */ false
+      );
+    } catch (error) {
+      console.error(error);
```

This console was helpful in debugging.  Errors were swallowed otherwise.